src/server/debug/server/server_test.go

```diff
@@ -208,7 +220,7 @@ func TestQueryLoki(t *testing.T) {
 			buildWant: func() []int {
 				var want []int
 				want = append(want, -2)
-				for i := 0; i < 4999; i++ {
+				for i := 0; i < serverMaxLogs-1; i++ {
```

Comment:
I actually don't understand why this has to be 1 less than the `serverMaxLogs`

Reply:
Log 1 is "-2", logs 2-serverMax are "-1", and then we seek over 40,000-ish lines and just return "0".  We seek over them because there is no way to address them with the Loki API, so we expect serverMaxLogs + 1 log.  It's the best we can do with their API :(